genesis/engine/solvers/rigid/collider_decomp.py

```diff
@@ -1,16 +1,18 @@
 import time
 import numpy as np
+import numpy.typing as npt
 import taichi as ti
 
 import genesis as gs
+from genesis.engine.solvers.rigid.rigid_solver_decomp import RigidSolver
 import genesis.utils.geom as gu
 from genesis.styles import colors, formats
 
 from .mpr_decomp import MPR
 
 
 @ti.func
-def rotaxis(vecin, i0, i1, i2, f0, f1, f2):
+def rotaxis(vecin: ti.Vector, i0: int, i1: int, i2: int, f0: float, f1: float, f2: float) -> ti.Vector:
```

Comment:
Plain int is fine? Instead of some Taichi-specific int32 / int34 ?

Reply:
I'll remove these. I was intending not to touch taichi funcs and kernels for now.

Reply:
(removed)